scripts/vm/hypervisor/xenserver/xenserver70/patch

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Thanks Koushik for the review. I have removed this file and am using the 6.5 one.